pyproject.toml

```diff
@@ -6,7 +6,7 @@ dynamic = ["version", "readme"]
 license={file="LICENCE"}
 
 dependencies = [
-    "ocf_data_sampler==0.0.26",
+    "ocf_data_sampler==0.0.32",
```

Comment:
This needs updating to a new version after https://github.com/openclimatefix/ocf-data-sampler/pull/82/files is merged in, should fix the failing test